index.html

```diff
@@ -224,16 +224,18 @@ <h3>legend</h3>
                 "searchable": true
               },
               {
+                "name": "mozPosition",
                 "data": "mozPosition",
                 "type": "enum-0",
                 "render": function (data, type, row, meta) {
                   if (type !== "display") {
                     return data;
                   }
                   if (data !== "" && ! status_styles.hasOwnProperty(data)) {
-                    console.log("WARNING: unrecognised position '" + data + "' on " + row.name)
+                    console.log("WARNING: unrecognised position '" + data + "' on " + row.name);
                   }
-                  return "<button type='button' class='btn btn-" + status_styles[data] + " btn-xs'>" + data + "</button>"
+                  return "<button type='button' class='btn btn-xs position btn-" + status_styles[data] +
+                    "' onclick='filterPosition'>" + data + "</button>";
```

Comment:
```suggestion
                    "' onclick='filterPosition(event)'>" + data + "</button>";
```

Reply:
That's my mistake; I should have reverted this one.  I'm using a different (better?) technique to attach the event handler below.

```suggestion
                    "'>" + data + "</button>";
```